Configurations/descrip.mms.tmpl

```diff
@@ -788,11 +784,14 @@ $bin.EXE : $deps
                      search_severity = \$severity
         @ ! search_severity is 3 when the last search didn't find any matching
         @ ! string: %SEARCH-I-NOMATCHES, no strings matched
-        -@ IF search_severity .NE. 3 .OR. link_severity .NE. 1 THEN -
-                TYPE $bin.LINKLOG
+        @ ! If that was the result, we pretend linking got through without
+        @ ! fault or warning.
+        @ IF search_severity .EQ. 3 THEN link_severity = 1
+        @ ! At this point, if link_severity shows that there was a fault
+        @ ! or warning, make sure to restore the linking status.
+        -@ IF .NOT. link_severity THEN TYPE $bin.LINKLOG
```

Comment:
For those who do not know: truth and falsehood in DCL (the standard VMS shell) is determined by the least significant bit of a number, all other bits are ignored.  So 0, 2, 4, ... are false and 1, 3, 5, ... are true,

This goes with the status codes (exit codes in Unix speak) where the 3 lowest bits constitute the severity and have the following meanings:

- 0: Warning (which is "not so successful")
- 1: Success
- 2: Error
- 3: Information (not a failure but a notification)
- 4: Fatal error

So the expression `.NOT. link_severity` simply checks if the status code shows success (1 or 3) or failure (0, 2, 4)